src/Traits/RendersImages.php

```diff
@@ -178,4 +178,45 @@ protected function renderBackground(): void
             $filledRows++;
         }
     }
+
+    protected function renderBackgroundURL(): void
```

Comment:
This looks like a straight copy of `renderBackground` with a few modifications, but the way this version works, I'm not sure the loop for repeating the background image is required.